packages/okta-react/package.json

```diff
@@ -4,17 +4,17 @@
   "description": "React support for Okta",
   "main": "./dist/index.js",
   "scripts": {
-    "build": "rimraf dist/ && babel src -d dist",
-    "build:harness": "npm --prefix test/e2e/harness install",
+    "build": "rimraf dist/ && yarn build:package-info && babel src -d dist",
```

Comment:
What is the reason for this change? Should we do that also for the other frameworks?

Reply:
Good question - This mirrors what we have with `okta-angular`, where we run `build:package-info` before we compile it.

This change keeps us in sync